src/godocToMarkdown.ts

```diff
@@ -50,24 +52,25 @@ const matchRx = '(' + urlRx + ')|(' + identRx + ')';
 // and the word is converted into a link. If nice is set, the remaining text's
 // appearance is improved where it makes sense (e.g., `` is turned into &ldquo;
 // and '' into &rdquo;).
-function emphasize(line: string, words: {string: string}, nice: boolean): string {
+const emphasize = (line: string, words: { [key: string]: string }, nice: boolean): string => {
 	const parts = [];
 
 	const regexp = XRegExp(matchRx, 'gu');
 	while (true) {
 		let m = XRegExp.exec(line, regexp);
-		if (m == null) {
+		if (m === null) {
```

Comment:
You need to check for `undefined` as well here

Reply:
not really, it always returns null on no match